bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/model/AppMode.scala

```diff
@@ -1,10 +1,19 @@
 package edu.gemini.pit.model
 
-object AppMode {
+import java.util.logging.Logger
+
+import scala.util.Try
 
-  lazy val isTAC = isDefined("edu.gemini.pit.tac")
-  lazy val isTest = isDefined("edu.gemini.pit.test")
+object AppMode {
+  private val Log = Logger.getLogger(AppMode.getClass.getName)
 
-  private def isDefined(prop:String) = Option(System.getProperty(prop)).isDefined
+  lazy val isTAC  = isDefinedAndTrue("edu.gemini.pit.tac")
```

Comment:
It certainly isn't set anywhere in the code, so yes, it would have to be a user option.
It looks to be used in the `UndoAction` and `RedoAction`, though, so I was hesitant to make any changes to it.